packages/paste-core/components/form/stories/select.stories.tsx

```diff
@@ -0,0 +1,315 @@
+      </>
+    );
+  })
+  .add('Select - Default Value', () => {
+    const uid = useUID();
+    const [value, setValue] = React.useState('');
+    return (
+      <>
+        <FormLabel htmlFor={uid}>Label</FormLabel>
+        <Select
+          id={uid}
+          onChange={event => {
+            setValue(event.target.value);
+            action('handleChange');
+          }}
+          onFocus={action('handleFocus')}
+          onBlur={action('handleBlur')}
+          defaultValue="option-3"
+          value={value}
+        >
+          <Option value="option-1">Option 1</Option>
+          <Option value="option-2">Option 2</Option>
+          <Option value="option-3">Option 3</Option>
+          <Option value="option-4">Option 4</Option>
+        </Select>
+        <FormHelpText>Info that helps a user with this field.</FormHelpText>
+      </>
+    );
+  })
+  .add('Select - Insert before and after', () => {
+    const uid = useUID();
+    const [value, setValue] = React.useState('Select - Error');
+    return (
+      <>
+        <FormLabel htmlFor={uid}>Label</FormLabel>
+        <Select
+          id={uid}
+          onChange={event => {
+            setValue(event.target.value);
+            action('handleChange');
+          }}
+          onFocus={action('handleFocus')}
+          onBlur={action('handleBlur')}
+          insertBefore={<div>$10.99</div>}
+          insertAfter={
+            <Anchor href="#" display="flex">
+              <InformationIcon decorative={false} title="Get more info" />
+            </Anchor>
+          }
+          value={value}
+        >
+          <Option value="option-1">Option 1</Option>
+          <Option value="option-2">Option 2</Option>
+          <Option value="option-3">Option 3</Option>
+          <Option value="option-4">Option 4</Option>
+        </Select>
+        <FormHelpText>Info that helps a user with this field.</FormHelpText>
+      </>
+    );
+  })
+  .add('Select - Option Groups', () => {
+    const uid = useUID();
+    const [value, setValue] = React.useState('Select - Error');
+    return (
+      <>
+        <FormLabel htmlFor={uid}>Label</FormLabel>
+        <Select
+          id={uid}
+          onChange={event => {
+            setValue(event.target.value);
+            action('handleChange');
+          }}
+          onFocus={action('handleFocus')}
+          onBlur={action('handleBlur')}
+          value={value}
+        >
+          <OptionGroup label="Group 1">
+            <Option value="option-1">Option 1</Option>
+            <Option value="option-2">Option 2</Option>
+          </OptionGroup>
+          <OptionGroup label="Group 2">
+            <Option value="option-3">Option 3</Option>
+            <Option value="option-4">Option 4</Option>
+          </OptionGroup>
+        </Select>
+        <FormHelpText>Info that helps a user with this field.</FormHelpText>
+      </>
+    );
+  })
+  .add('Select - Option Groups and Multiple', () => {
```

Comment:
Running this story local, I can't seem to select an option.

Reply:
Yeah I just noticed that too, I broke something with the `onChange` handler for the story when I was adjusting the type interface. Will fix. Thank you for pointing this out!